### ISSUE TITLE
[Brewmaster_Beta] Package conflict with libsdl1.2-dev

### ISSUE THREAD

**Issue by ProfessorKaos64** [ValveSoftware/SteamOS#335]
Using the Jessie repos, I received the following conflict:
```
==> Could not install package libsdl1.2-dev from jessie repo, trying jessie-backports
Reading package lists...
Building dependency tree...
Reading state information...
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 libsdl1.2-dev : Depends: libdirectfb-dev (>= 0.9.22) but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
```
This is part of my desktop-software script [here](https://github.com/ProfessorKaos64/SteamOS-Tools/blob/brewmaster-beta/desktop-software.sh). Repo sources are added via [add-debian-repos](https://github.com/ProfessorKaos64/SteamOS-Tools/blob/brewmaster-beta/add-debian-repos.sh).

I know desktop mode is not "supported," but this worked fine in Alchemist, so I suspect there is a version / missing package issue, or something I can't see here. This is required for me to buid [Retroarch](https://github.com/libretro/RetroArch) from source.

**Comment by johnv-valve** [comment]
libjpeg-turbo is missing from the brewmaster repositories. If you follow the "is not going to be installed" chain, you'll eventually find that libjpeg62-turbo-dev (from jessie) won't install because brewmaster has a newer version of libjpeg62-turbo already installed.

You can downgrade libjpeg62-turbo to the jessie version or wait for the fix in the next brewmaster release.

**Comment by ProfessorKaos64** [comment]
@johnv-valve , is this thread in the Steam Universe forum or here in the issues listings? I will book mark it for review for these kind of things. Sorry for the report then.

**Comment by johnv-valve** [comment]
This is fixed in the current brewmaster_beta (2.20)

I don't think there is any thread in the forum on this. I actually prefer having these kind of issues reported here on github since it is easier to track.

**Comment by ProfessorKaos64** [comment]
```
I don't think there is any thread in the forum on this. I actually prefer having these kind of issues reported here on github since it is easier to track.
```
That was my reasoning here. I'll try to keep this in mind when I have something come up @johnv-valve . Than you for getting to this.
